v4.6.2 suddenly fails with cannot unmarshal object into Go struct field _UserGetSingleton.type of type string

<!---
Please note the following potential times when an issue might be in Terraform core:

* [Configuration Language](https://www.terraform.io/docs/configuration/index.html) or resource ordering issues
* [State](https://www.terraform.io/docs/state/index.html) and [State Backend](https://www.terraform.io/docs/backends/index.html) issues
* [Provisioner](https://www.terraform.io/docs/provisioners/index.html) issues
* [Registry](https://registry.terraform.io/) issues
* Spans resources across multiple providers

If you are running into one of these scenarios, we recommend opening an issue in the [Terraform core repository](https://github.com/hashicorp/terraform/) instead.
--->

<!--- Please keep this note for the community --->

### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

### Terraform Version

```
terraform -v
Terraform v1.5.5
on darwin_arm64
+ provider registry.terraform.io/hashicorp/aws v5.26.0
+ provider registry.terraform.io/hashicorp/external v2.3.2
+ provider registry.terraform.io/hashicorp/vault v3.23.0
+ provider registry.terraform.io/okta/okta v4.6.2
```

### Affected Resource(s)

<!--- Please list the affected resources and data sources. --->

- okta provider configuration

### Terraform Configuration Files

<!--- Information about code formatting: https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code --->

```hcl
provider "okta" {
  api_token = var.okta_api_token
  base_url  = "okta.com"
  org_name  = "my_org"
}
```

### Debug Output

```
Planning failed. Terraform encountered an error while generating this plan.

╷
│ Error: [ERROR] failed validate configuration: error with v3 SDK client: json: cannot unmarshal object into Go struct field _UserGetSingleton.type of type string
│
│   with provider["registry.terraform.io/okta/okta"],
│   on main.tf line 25, in provider "okta":
│   25: provider "okta" {
```

### Expected Behavior

During plan/apply, the provider should be able to configure itself like it did with v4.6.1 using the same configuration.

### Actual Behavior

The provider errors with the above output.

### Steps to Reproduce

1. Use Okta v4.6.2
2. `terraform apply`

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,11 @@
 # Changelog
 
+## 4.6.3 (November 29, 2023)
+
+### BUG FIXES
+
+*  Hot fix due to breaking change in okta-sdk-golang [#1838](https://github.com/okta/terraform-provider-okta/pull/1838). Thanks, [@duytiennguyen-okta](https://github.com/duytiennguyen-okta)!
+
 ## 4.6.2 (November 28, 2023)
 
 ### BUG FIXES

--- a/go.mod
+++ b/go.mod
@@ -20,7 +20,7 @@ require (
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.30.0
 	github.com/jarcoal/httpmock v1.3.1
 	github.com/kelseyhightower/envconfig v1.4.0
-	github.com/okta/okta-sdk-golang/v3 v3.0.18
+	github.com/okta/okta-sdk-golang/v3 v3.0.19
 	github.com/patrickmn/go-cache v2.1.0+incompatible
 	github.com/stretchr/testify v1.8.4
 	gopkg.in/dnaeon/go-vcr.v3 v3.1.2

--- a/okta/config.go
+++ b/okta/config.go
@@ -25,7 +25,7 @@ import (
 )
 
 const (
-	OktaTerraformProviderVersion   = "4.6.2"
+	OktaTerraformProviderVersion   = "4.6.3"
 	OktaTerraformProviderUserAgent = "okta-terraform/" + OktaTerraformProviderVersion
 )
 

--- a/website/docs/index.html.markdown
+++ b/website/docs/index.html.markdown
@@ -24,7 +24,7 @@ terraform {
   required_providers {
     okta = {
       source = "okta/okta"
-      version = "~> 4.6.2"
+      version = "~> 4.6.3"
     }
   }
 }

